src/Analyzers/Core/CodeFixes/FileHeaders/AbstractFileHeaderCodeFixProvider.cs

```diff
@@ -167,8 +176,12 @@ private SyntaxNode ReplaceHeader(Document document, SyntaxNode root, string expe
 
         private SyntaxNode AddHeader(Document document, SyntaxNode root, string expectedFileHeader)
         {
+#if CODE_STYLE
```

Comment:
Same here, this can be reverted.

Reply:
➡️ Keeping this discussion on https://github.com/dotnet/roslyn/pull/42220#discussion_r389069003